openpype/tools/utils/delegates.py

```diff
@@ -30,9 +30,11 @@ def __init__(self, dbcon, *args, **kwargs):
     def displayText(self, value, locale):
         if isinstance(value, HeroVersionType):
             return lib.format_version(value, True)
-        assert isinstance(value, numbers.Integral), (
-            "Version is not integer. \"{}\" {}".format(value, str(type(value)))
-        )
+        if not isinstance(value, numbers.Integral):
+            # For cases where no version is resolved like NOT FOUND cases
+            # where a representation might not exist in current database
+            return
```

Comment:
Maybe we should display something?
```suggestion
            return "N/A"
```

Reply:
I'll give it a go next week and take a screenshot of both empty column and N/A in column to see what looks best.

Side note: To me _not applicable_ doesn't sound entirely right because it's more a case of "was unable to resolve but it should have been applicable". :) It would be more like "unknown", "unidentified", "undetermined".

Reply:
> To me not applicable doesn't sound entirely right because it's more a case of "was unable to resolve but it should have been applicable". :) It would be more like "unknown", "unidentified", "undetermined".

Too long for version column.

Reply:
> Too long for version column.

Agreed. Was just trying to find the right terminology

---

Here's a visual comparsion.

**Empty version column**

![afbeelding](https://user-images.githubusercontent.com/2439881/226307599-5276bf8d-dafc-4c01-b38b-e3cca9b6b087.png)

**Version column with N/A**

![afbeelding](https://user-images.githubusercontent.com/2439881/226307923-86cc4a92-1d6c-4a89-a95c-6ffff65156e3.png)

---

I personally prefer the empty/cleaner one without N/A but would love to hear more opinions.

Reply:
@mkolar @antirotor opinions? It can be easily changed when needed in future.

Reply:
I think the empty is better because it is obvious from the first glance. Maybe even hyphen `-` could be used because it is visually different than any letters/numbers, but empty works for me.